### PR TITLE
(spanish): 2-Regression/3-Linear/README.md

### PR DESCRIPTION
**Pull request by hereje**
Translate file `2-Regression/3-Linear/README.md` to spanish

### CODE REVIEW

File: 2-Regression/3-Linear/translations/README.es.md [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+
+Ahora, debes estar familiarizado con la estructura de los datos de las calabazas que ya examinamos. Puedes encontrarlos precargados y pre-limpiados en el archivo _notebook.ipynb_ de esta lección. En el archivo, el precio de la calabaza se muestra por fanega en un nuevo dataframe. Asegúrate que puedas ejecutar estos notebooks en kernels en Visual Studio Code.
+
+### Preparación
+
+Como recordatorio, estás cargando estos datos para hacer preguntas aceca de estos.
+
+- ¿Cuándo es el mejor momento para comprar calabazas?
+- ¿Qué precio puedo esperar para el caso de calabazas miniatura?
+- ¿Debería comprarlas en cestos de media fanega o por caja de 1 1) de fanega?
```
Comment (alfredodeza) [inline 2-Regression/3-Linear/translations/README.es.md:28]: This looks like an extra number and a parenthesis that is not needed

File: 2-Regression/3-Linear/translations/README.es.md [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+
+Sigamos profundizando en estos datos.
+
+En la lección anterior, creaste un dataframe de Pandas y lo poblaste con parte del conjunto de datos original, estandarizando el precio de la fanega. Haciéndolo, sólo fuiste capaz de reunir alrededor de 400 puntos de datos y sólo para los meses de otoño.
+
+Da un vistazo a los datos que fueron precargados en el notebook que acompaña a esta lección. Los datos están precargados y hay graficada un gráfico de dispersión inicial para mostrar datos mensuales. Quizá podamos obtener un poco más de detalle acerca de la naturaleza de los datos limpiándolos más.
```
Comment (alfredodeza) [inline 2-Regression/3-Linear/translations/README.es.md:34]: Seems redundant. Feel free to adjust or use my recommendation

```suggestion
Da un vistazo a los datos que fueron precargados en el notebook que acompaña a esta lección. Los datos están precargados con un gráfico de dispersión inicial para mostrar datos mensuales. Quizá podamos obtener un poco más de detalle acerca de la naturaleza de los datos limpiándolos más.
```

File: 2-Regression/3-Linear/translations/README.es.md [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+## Un línea de regresión lineal
+
+Como aprendiste en la lección 1, el objetivo de un ejercicio de regresión lineal es ser capaz de graficar una línea para:
+
+- **Mostrar la relación de las variables**. Mostrar la relación entre las variables
+- **Realizar predicciones**. Hacer predicciones precisas en donde un nuevo punto de datos caería en relación a esa línea.
+
+Es típico de la **regresión de mínimos cuadrados** el dibujar este tipo de línea. El término 'mínimos cuadrados' significa que todos los puntos de datos rodeando la línea de regresión se elevan al cuadrado y luego se suman. Idealmente, la suma final es tan pequeña como sea posible, porque queremos un número bajo de errores, o `mínimos cuadrados`.
+
+Lo hacemos así ya que que queremos modelar una línea que tiene la distancia acumulada menor de todos nuestros puntos de datos. Tambień elevamos al cuadraro los términos antes de sumarlos ya que nos interesa su magnitud en lugar de su direción.
```
Comment (alfredodeza) [inline 2-Regression/3-Linear/translations/README.es.md:45]:
```suggestion
Lo hacemos así ya que queremos modelar una línea que tiene la menor distancia acumulada de todos nuestros puntos de datos. También elevamos al cuadrado los términos antes de sumarlos ya que nos interesa su magnitud en lugar de su dirección.
```

File: 2-Regression/3-Linear/translations/README.es.md [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+
+> **🧮 Muéstrame las matemáticas**
```
Comment (alfredodeza) [inline 2-Regression/3-Linear/translations/README.es.md:47]: Why is this section using `>` ?

File: 2-Regression/3-Linear/translations/README.es.md [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
+>
+> Calcula el valor de Y. ¡Si estás pagando alrededor de $4, debe ser Abril! Infografía de [Jen Looper](https://twitter.com/jenlooper)
+>
+> Las matemáticas que calculan la línea deben demostrar la pendiente de la línea, la cual también depende de la intercepción, o dónde `Y` se sitúa cuando `X = 0`.
+>
+> Puedes observar el método de cálculo para estos valores en el sitio web [las matemáticas son divertidas](https://www.mathsisfun.com/data/least-squares-regression.html). También visita esta [calculadora de mínimos cuadrados](https://www.mathsisfun.com/data/least-squares-calculator.html) para ver cómo los valores de los números impactan la línea.
+
+## Correlación
+
+Un término más a entender es el **coeficiente de correlación** entre las variables dadas X e Y. Usando un gráfico de dispersión, puedes visualizar rápidamente este coeficiente. Un gráfico con pountos de datos dispersos en una línea ordenada tienen alta correlación, pero una gráfico con puntos de datos dispersos por todas partes entre X e Y tienen baja correlación.
```
Comment (alfredodeza) [inline 2-Regression/3-Linear/translations/README.es.md:73]:
```suggestion
Un término más a entender es el **coeficiente de correlación** entre las variables dadas X e Y. Usando un gráfico de dispersión, puedes visualizar rápidamente este coeficiente. Un gráfico con puntos de datos dispersos en una línea ordenada tienen alta correlación, pero un gráfico con puntos de datos dispersos por todas partes entre X e Y tienen baja correlación.
```

File: 2-Regression/3-Linear/translations/README.es.md [hunk 6]
```diff
@@ -0,0 +1,341 @@
...
+
+Un buen modelo de regresión lineal será aquél que tenga un alto Coeficiente de Correlación (más cercano a 1 que a 0) usando el métro de regresión de mínimos cuadrados con una línea de regresión.
+
+✅ Ejecutta el notebook que acompaña esta lección y mira el gráfico de Ciudad a Precio. ¿Los datos asociados de Ciudad a Precio para las ventas de calabaza parecen tener correlación lata o baja, de acuerdo a tu intepretación visual del gráfico de dispersión?
```
Comment (alfredodeza) [inline 2-Regression/3-Linear/translations/README.es.md:77]:
```suggestion
✅ Ejecuta el notebook que acompaña esta lección y mira el gráfico de Ciudad a Precio. ¿Los datos asociados de Ciudad a Precio para las ventas de calabaza parecen tener correlación alta o baja, de acuerdo a tu interpretación visual del gráfico de dispersión?
```

File: 2-Regression/3-Linear/translations/README.es.md [hunk 7]
```diff
@@ -0,0 +1,341 @@
...
+
+## Prepara tus datos para la regresión
+
+Ahora que tienes conocimiento de las matemáticas detrás de este ejercicio, crea un modelo de regresión para ver si puedes predecir cuál de los paquetes de calabazas tendrá los mejores precios. Alguien comprando calabazas para una parcela de calabazas de días festivos quisiera esta información para ser capaz de optimizar sus compras de paquetes de calabazas para la parcela.
```
Comment (alfredodeza) [inline 2-Regression/3-Linear/translations/README.es.md:81]:
```suggestion
Ahora que tienes conocimiento de las matemáticas detrás de este ejercicio, crea un modelo de regresión para ver si puedes predecir cuál de los paquetes de calabazas tendrá los mejores precios. Alguien comprando calabazas para una parcela de calabazas en días festivos quisiera esta información para ser capaz de optimizar sus compras de paquetes de calabazas para la parcela.
```

File: 2-Regression/3-Linear/translations/README.es.md [hunk 8]
```diff
@@ -0,0 +1,341 @@
...
+
+Ya que usarás Scikit-learn, no hay razón para hacer esto a mano (¡aunque podrías!). En el bloque principal de procesamientos de datos de tu notebook de lección, agrega una biblioteca de Scikit-learn para convertir automáticamente todos los datos de cadena a números:
+
+```python
+from sklearn.preprocessing import LabelEncoder
+
+new_pumpkins.iloc[:, 0:-1] = new_pumpkins.iloc[:, 0:-1].apply(LabelEncoder().fit_transform)
+```
+
+Si ahora miras el nuevo dataframe new_pumpkins, ves que todas las cadenas ahora son numéricas. ¡Esto te dificulta el leer pero lo hace más comprensible para Scikit-learn!
```
Comment (alfredodeza) [inline 2-Regression/3-Linear/translations/README.es.md:91]:
```suggestion
Si ahora miras el nuevo dataframe `new_pumpkins`, ves que todas las cadenas ahora son numéricas. ¡Esto te dificulta el leer pero lo hace más comprensible para Scikit-learn!
```

File: 2-Regression/3-Linear/translations/README.es.md [hunk 9]
```diff
@@ -0,0 +1,341 @@
...
+Ahora puedes tomar decisiones más informadas (no sólo basado en ver un gráfico de dispersión) acerca de los datos que mejor se ajustan a la regresión.
```
Comment (alfredodeza) [inline 2-Regression/3-Linear/translations/README.es.md:92]:
```suggestion
Ahora puedes tomar decisiones más informadas (no sólo basado en un gráfico de dispersión) acerca de los datos que mejor se ajustan a la regresión.
```

File: 2-Regression/3-Linear/translations/README.es.md [hunk 10]
```diff
@@ -0,0 +1,341 @@
...
+
+Intenta encontrar una buena correlación entre dos puntos de tus datos para construir potencialmente un buen modelo predictivo. Como resulta, sólo hay correlación débil entre las Ciudad y el Precio.
```
Comment (alfredodeza) [inline 2-Regression/3-Linear/translations/README.es.md:94]:
```suggestion
Intenta encontrar una buena correlación entre dos puntos de tus datos para construir potencialmente un buen modelo predictivo. Como resultado, sólo hay correlación débil entre las Ciudad y el Precio.
```

File: 2-Regression/3-Linear/translations/README.es.md [hunk 11]
```diff
@@ -0,0 +1,341 @@
...
+
+```python
+print(new_pumpkins['City'].corr(new_pumpkins['Price']))
+0.32363971816089226
+```
+
+sin embargo, existe una correlación un poco mejor entre el Paquete y su Precio. Esto tiene sentido, ¿cierto? Normalmente, entre más grande sea la caja producidad, mayor será el precio.
```
Comment (alfredodeza) [inline 2-Regression/3-Linear/translations/README.es.md:101]:
```suggestion
Sin embargo, existe una correlación un poco mejor entre el Paquete y su Precio. Esto tiene sentido, ¿cierto? Normalmente, entre más grande sea la caja producida, mayor será el precio.
```

**Review by alfredodeza — CHANGES_REQUESTED** [review]
Looks like there is a lot of work needed for spelling/grammar. I stopped half-way through. Could you use a spell checker or something similar to catch as many as possible? I also added some other non-spelling suggestions

File: 2-Regression/3-Linear/translations/README.es.md (hunk 4 above)
Comment (hereje) [reply 2-Regression/3-Linear/translations/README.es.md:47]: Honestly, I don't know. The original file contains the same markup.

**Comment by jlooper** [comment]
Is this file ready to be merged? Thanks for the review @alfredodeza

File: 2-Regression/3-Linear/translations/README.es.md (hunk 7 above)
Comment (alfredodeza) [reply 2-Regression/3-Linear/translations/README.es.md:81]: Seems like this one didn't get updated

**Review by alfredodeza — APPROVED** [review]

**Pull request merged**